packages/@okta/vuepress-site/guides/custom-url-domain/enable-the-custom-domain/index.md

```diff
@@ -39,19 +39,16 @@ Okta serves traffic over HTTPS (TLS) on your custom domain. Use this section to
 
 ### Create a CNAME Record for Your Subdomain
 Before Okta can serve traffic over your domain, you need to add an alias from your custom domain to the Okta subdomain of your Okta organization. You do this by creating or modifying a CNAME record for your custom domain name.
+    
+1. Return to your Domain Name registrar and locate the option to modify your DNS records.
+1. Create a new CNAME record and paste the value from the **Host** column into your registrar's **Name** or **Host** field.
 
-1. Copy the values of the **Host** and **Data** columns of the **CNAME** table in Okta into a text file. For example, depending on what type of Okta org you have, the **Data** value should look similar to the following:
+> Important: Depending on your registrar, you may only need to enter the subdomain part. For example, if you picked the subdomain `id.example.com`, your registrar may only require you to create a CNAME record for `id`. If you're not sure, check your registrar's documentation.
 
-    - `org.Subdomain.customdomains.okta.com`
-    - `org.Subdomain.customdomains.oktapreview.com`
-    - `org.Subdomain.customdomains.okta-emea.com`
-    
-2. Return to your Domain Name registrar and locate the option to modify your DNS records.
-3. Create a CNAME record and paste the value that you copied from the **Host** column into the appropriate field, for example, the **Name** or **Host** field. 
-4. Paste the value that you copied from the **Data** column into the appropriate field, for example, the **Record** or **Value** field.
-5. Save the record.
+1. Paste the value from the **Data** column into your registrar's **Record** or **Value** field.
```

Comment:
Will it always be a **Record** or **Value** field at their registrar's? Do we know for sure? If so, I'm okay leaving out "for example", otherwise, "for example" implies that it MAY be called **Record** or **Value**, but it MAY be called something else. Your thoughts?

Reply:
It MAY be called something else, yes. Different registrars have different UI so I'm not certain it will always be called that.

Reply:
So, how about this:

Paste the value from the **Data** column into the appropriate field at the registrar, for example, the **Record** or **Value** field.